src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -244,6 +271,21 @@ public boolean perform(final AbstractBuild<?, ? > build, final Launcher launcher
 
             reportStart(listener, subJob);
             List<Action> actions = new ArrayList<Action>();
+
+            if (resume) {
+                SubBuild subBuild = failedBuildMap.get(subJob.getUrl());
+                if (null != subBuild) {
+                    Item item = Jenkins.getInstance().getItem(subBuild.getJobName(), multiJobBuild.getParent(),

```

Comment:
With folders it works fine, but now I see other problems with folders in this plugin. May create issue after...